crates/gitbutler-core/Cargo.toml

```diff
@@ -8,6 +8,7 @@ publish = false
 [dev-dependencies]
 once_cell = "1.19"
 pretty_assertions = "1.4"
+gitbutler-testsupport = { path = "../gitbutler-testsupport" }
```

Comment:
I've been putting crate deps in the workspace Cargo.toml and then using `.workspace = true` - is it cool if we follow that convention?

Reply:
Thanks for letting me know, I will remember that. It's fixed via 4542c73fdb4ab51c14ca03ae036d869e00e482d5.